### PR TITLE
[RHCLOUD-33579] Cleanup Ansible templates and initiate the new one

### PR DESCRIPTION
**Pull request by g-duval**

### CODE REVIEW

File: engine/src/main/resources/templates/AnsibleServiceOnAws/notifyCustomerProvisionSuccessEmailBodyV2.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{@boolean renderSection1=true}
+{@boolean renderButtonSection1=true}
+{#include Common/insightsEmailBody}
+{#content-title}
+    Ansible Automation Platform Service on AWS - Ansible Automation Platform
+{/content-title}
+{#content-title-section1}
+    Notify customer environment is ready
```
Comment (MikelAlejoBR) [inline engine/src/main/resources/templates/AnsibleServiceOnAws/notifyCustomerProvisionSuccessEmailBodyV2.html:8]: Wouldn't this be something like "Environment is ready"?

**Review by MikelAlejoBR — APPROVED** [review]
Just a minor comment regarding the wording of the section title and the header for the emails.

File: engine/src/main/resources/templates/AnsibleServiceOnAws/notifyCustomerProvisionSuccessEmailBodyV2.html (hunk 1 above)
Comment (g-duval) [reply engine/src/main/resources/templates/AnsibleServiceOnAws/notifyCustomerProvisionSuccessEmailBodyV2.html]: You are right, that's better, even for a draft :)

File: engine/src/main/resources/templates/AnsibleServiceOnAws/notifyCustomerProvisionSuccessEmailTitleV2.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Instant notification - Notify customer environment is ready - Ansible Automation Platform Service on AWS - Ansible Automation Platform
```
Comment (MikelAlejoBR) [inline engine/src/main/resources/templates/AnsibleServiceOnAws/notifyCustomerProvisionSuccessEmailTitleV2.txt:1]:
```suggestion
Instant notification - Environment is ready - Ansible Automation Platform Service on AWS - Ansible Automation Platform
```

File: engine/src/main/resources/templates/AnsibleServiceOnAws/notifyCustomerProvisionSuccessEmailBodyV2.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+{#content-title}
+    Ansible Automation Platform Service on AWS - Ansible Automation Platform
+{/content-title}
+{#content-title-section1}
+    Environment is ready
+{/content-title-section1}
+{#content-body-section1}
+    <p>Requested environment '{action.context.env_name}' is ready.</p>
+{/content-body-section1}
+{/include}
```
Comment (MikelAlejoBR) [inline engine/src/main/resources/templates/AnsibleServiceOnAws/notifyCustomerProvisionSuccessEmailBodyV2.html:13]: Missing new line!

File: engine/src/main/resources/templates/AnsibleServiceOnAws/notifyCustomerProvisionSuccessEmailTitleV2.txt (hunk 2 above)
Comment (g-duval) [reply engine/src/main/resources/templates/AnsibleServiceOnAws/notifyCustomerProvisionSuccessEmailTitleV2.txt:1]: It supposed to be a fix format with "Instant notification - EventTypeDisplayName - ApplicationDisplaName - BundleName", I prefer let it as it is and update it later regarding what Ansible team will decide if it's ok for you?

Comment (MikelAlejoBR) [reply engine/src/main/resources/templates/AnsibleServiceOnAws/notifyCustomerProvisionSuccessEmailTitleV2.txt:1]: Ah yeah, that's fine then!

**Pull request merged**